### PR TITLE
Cog 281 Terminology Change for RACFID to CWS Login 

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**

### CODE REVIEW

File: lib/users/verify_user.rb [hunk 1]
```diff
@@ -37,6 +37,7 @@ class VerifyUser < Dry::Struct
     attribute :expires, Types::String.optional
     attribute :location, Types::String.optional
     attribute :pragma, Types::String.optional
+    attribute :error_code, Types::String.optional
```
Comment (RamuKammagani-osi) [inline lib/users/verify_user.rb]: missing specs!

Comment (Sushmareddy-OSI) [reply lib/users/verify_user.rb]: done!

**Comment by eckdanny-osi** [comment]
@Sushmareddy-OSI #103 (merged) encapsulates the _tweaking_ we had to do to get Acceptance Tests passing again. Merge `master` into your PR please and we should get some helpful CI checks again.

File: app/javascript/_utils/errorCodes.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export function verificationError(errorCode, email, racfid) {
```
Comment (eckdanny-osi) [inline app/javascript/_utils/errorCodes.js]: I think the signature for this fn should be 

```ts
(errCode: String, options: Map<{ [key: String]: any }> = {}): String
```

positional arguments will get quite messy as more cases are written

**Comment by Sushmareddy-OSI** [comment]
from Java API we are getting response as 
`{
  "verification_passed": false,
  "verification_message": "No user with RACFID: THOMALA is found in CWSCMS",
  "error_code": "CAP-003"
}`
we don't want the verification message using word `RACFID`. We are manipulating the  verification_message based on error_code.

File: app/javascript/_utils/errorCodes.js (hunk 2 above)
Comment (kelfink) [reply app/javascript/_utils/errorCodes.js]: Looks like a small enough change.  Should help us future-proof the code.

File: app/javascript/views/addUser/AddVerifiedUser.test.js [hunk 3]
```diff
@@ -84,14 +84,12 @@ describe('AddVerifiedUser', () => {
       const newUserDetails = {
         verification_passed: false,
         verification_message: 'No ID',
+        error_code: 'CAP',
       };
       wrapper = shallow(
         <AddVerifiedUser verifyNewUserDetails={newUserDetails} />
       );
       expect(wrapper.find('Alert').length).toBe(1);
-      expect(wrapper.find('Alert').props().children).toBe(
```
Comment (kelfink) [inline app/javascript/views/addUser/AddVerifiedUser.test.js]: is there no message anymore?  This had ben verifying the text of the message, right?

Comment (Sushmareddy-OSI) [reply app/javascript/views/addUser/AddVerifiedUser.test.js]: @kelfink, method verificationError() in _utils/error_codes where we are overriding the message by error_codes. we are not using the message that is generated by java API

**Comment by kelfink** [comment]
Looks like this story is still necessary because the messages Perry spits out are not formatted for user consumption.  They are formatted for debugging or technical purposes. Users, however, are not expected to understand terms like RACFID.  SO the further formatting is needed.

**Pull request merged**